### DISCUSSION TITLE
Auto-login for floodgate players that linked their account using global linking

### DISCUSSION THREAD

**Discussion by Xandar-YT** [games647/FastLogin#1168] [General]
Is it possible for floodgate players that linked their bedrock account to their premium account using global linking to use auto-login?
I tried everything and every time it just asks for a password, even though I used /premium multiple times.
Thanks for any answers

**Comment by AydenCZR** [comment]
What plugin do you use to autologin? i use FastLogin to auto login and they have an option to autologin floodgate players that linked their bedrock account to their premium account using global linking.

autoLoginFloodgate: false --> linked

**Reply by AydenCZR** [reply]
its been a few months but i hope this still helps
